index.html

```diff
@@ -1,18 +1,18 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8">
     <title>Hello World!</title>
   </head>
   <body>
     <h1>Hello World!</h1>
     <!-- All of the Node.js APIs are available in this renderer process. -->
-    We are using node <script>document.write(process.versions.node)</script>,
-    Chromium <script>document.write(process.versions.chrome)</script>,
-    and Electron <script>document.write(process.versions.electron)</script>.
+    We are using node <script type="text/javascript">document.write(process.versions.node)</script>,
```

Comment:
i am just writing valid HTML Document that's it. and its not browser anymore. i have back experience when my script tag does not work so i add type and its working from then i always that attribute